### PR TITLE
Update find-bugs and find-builds for payload diffs

### PR DESCRIPTION
**Pull request by smunilla**
This will let us find bugs and builds that have changed between two image payloads.

### CODE REVIEW

**Comment by smunilla** [comment]
Supercedes #32

File: elliott [hunk 1]
```diff
@@ -377,34 +386,37 @@ advisory with the --add option.
...
 \b
-    $ elliott --group openshift-3.7 advisory:find-bugs --auto --flag bro_ok
+    $ elliott --group openshift-3.7 advisory:find-bugs --mode sweep --flag bro_ok
 
     Add two bugs to advisory 123456. Note that --group is not required
     because we're not auto searching:
 
 \b
-    $ elliott advisory:find-bugs --id 8675309 --id 7001337 --add 123456
+    $ elliott advisory:find-bugs --mode list --id 8675309 --id 7001337 --add 123456
```
Comment (tbielawa) [inline elliott]: `advisory:find-bugs`!!!!

**Comment by tbielawa** [comment]
with some doctext I give this
/lgtm

**Comment by tbielawa** [comment]
<3 docs. Thank youuuuuuuuu
/lgtm

**Pull request merged**